test/100-basic-name-resolution.bats

```diff
@@ -165,3 +165,26 @@ load helpers
 	# contain unexpected warning.
 	assert "$output" !~ "WARNING: recursion requested but not available"
 }
+
+# Internal network, meaning no DNS servers.
+# Hence all external requests must fail.
+@test "basic container - internal network has no DNS" {
```

Comment:
test seems logical, given the risk of regression we should also test the ipv6 path.